werk-vscode/LICENSE

```diff
@@ -0,0 +1,6 @@
+werk-vscode is dual-licensed under either
```

Comment:
`vsce` really wants a single license file present, otherwise it will print
```
 WARNING  LICENSE, LICENSE.md, or LICENSE.txt not found
Do you want to continue? [y/N] 
```